app/views/pages/home/_vacancy_facets.haml

```diff
@@ -0,0 +1,30 @@
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    %h3.govuk-heading-s= t('.cities')
+    %ul.govuk-list.two-column-list{ aria: { label: "#{t('.cities')} list" } }
+      - @cities_facets.first(20).each do |city, count|
```

Comment:
Are these sorted by number of hits?

Reply:
No. It made sense to me to sort them by number of hits and then alphabetically, but Chris said he just wants them to be alphabetically sorted.

Reply:
Hmm. I'm not sure about that - surely we need the top 20 cities with the most results sorted alphabetically?

Reply:
instead of the first(20) part shouldnt this be done elsewhere in a presenter or similar or better still when the data is queried?

Reply:
Was thinking I should probably bring this up at stand up.

Reply:
Have now changed it so it's sorted by count and then alphabetically.